hack/maybe-build-release.sh

```diff
@@ -16,3 +17,21 @@ make release VERSION=${tag}
 
 echo "Publish release ${tag}"
 make release-publish-images VERSION=${tag}
+
+echo "Tagging and pushing operator images to RedHat Connect for certification..."
+
+docker login -u unused scan.connect.redhat.com --password-stdin <<< ${OPERATOR_RH_REGISTRY_KEY}
+quayImage=quay.io/tigera/operator:$VERSION
+redhatImage=scan.connect.redhat.com/$OPERATOR_RH_PROJECTID/operator:$VERSION
+docker pull $quayImage
+docker tag $quayImage $redhatImage
+# Pushes to scan.connect.redhat.com fail if the image exists already. Return
+# a success exit code so we can retry image pushes if needed.
```

Comment:
I don't understand the comment. I don't see any retry being done.
If we're counting on the push failing if the image exists already, could we just check for it before doing the push?

Reply:
I updated the script as you suggested to check if the image exists